include/zephyr/devicetree.h

```diff
@@ -1189,8 +1189,8 @@
  *
  *     #define NODE DT_NODELABEL(n)
  *
- *     DT_LABEL(DT_PHANDLE_BY_NAME(NODE, io_channels, sensor))  // "ADC_1"
- *     DT_LABEL(DT_PHANDLE_BY_NAME(NODE, io_channels, bandgap)) // "ADC_2"
+ *     DT_PROP(DT_PHANDLE_BY_NAME(NODE, io_channels, sensor), foobar)  // "ADC_1"
```

Comment:
Suggestion, feel free to ignore: if you rename the adc nodes `adc1: adc@1` and `adc2: adc@2`, you could do `DT_NODE_FULL_NAME(DT_PHANDLE_BY_NAME(NODE, io_channels, sensor)) // "adc@1"`, and similarly for the other example. Then you could remove the `foobar` properties entirely.

Similarly for the other examples in here.

Reply:
I think accessing a prop in the node makes for a slightly more useful example for people.